gdi/opentelemetry/install-k8s-addon-eks.rst

```diff
@@ -1,34 +1,32 @@
 .. _install-k8s-addon-eks:
 
 *******************************************************************************
-Splunk Distribution of the OTel Collector Add-on for AWS EKS
+Splunk Distribution of the OpenTelemetry Collector as EKS Add-on 
```

Comment:
We should sate AWS in some form or manner on this page (Add-on for AWS EKS). I'll leave this up to the Docs team.

Reply:
On it!